package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "dev": "npm run build-scripts && next -p 3000",
     "build": "npm run build-scripts && next build",
+    "test":"",
```

Comment:
Why is it empty?

Reply:
It will fail test if i add jest in it because this PR doesn't have tests. I will jest in it whenever I open new PR

Reply:
We will have to do this only until the 1st PR gets merged